pypeit/spectrographs/vlt_fors.py

```diff
@@ -68,8 +68,8 @@ def default_pypeit_par(cls):
 
         # Sensitivity function parameters
         par['sensfunc']['algorithm'] = 'IR'
-        par['sensfunc']['polyorder'] = 8
-        par['sensfunc']['IR']['telgridfile'] = 'TelFit_Paranal_NIR_9800_25000_R25000.fits'
+        par['sensfunc']['polyorder'] = 5
+        par['sensfunc']['IR']['telgridfile'] = 'TelFit_Paranal_VIS_4900_11100_R25000.fits'
```

Comment:
I thought I already fixed this..

Reply:
I think the merge had some problems honestly. Some spooky things like this happened here and there.

Reply:
Fixed.